assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -11,33 +9,17 @@ import { Provider } from 'react-redux';
 import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
 import SimpleFooter from 'components/footers/simpleFooter/simpleFooter';
 
-import * as ga from 'helpers/ga';
-import * as abTest from 'helpers/abtest';
-import * as logger from 'helpers/logger';
+import * as pageStartup from 'helpers/pageStartup'
 import getQueryParameter from 'helpers/url';
 import Introduction from './components/Introduction';
 import Bundles from './components/Bundles';
 import WhySupport from './components/WhySupport';
 import WaysOfSupport from './components/WaysOfSupport';
 import reducer from './reducers/reducers';
 
-// ----- AB Tests ----- //
-
-const participation = abTest.init();
-
-
-// ----- Tracking ----- //
-
-ga.init();
-ga.setDimension('experience', abTest.getVariantsAsString(participation));
-ga.trackPageview();
-
-// ----- Logging ----- //
-
-logger.init();
+const participation = pageStartup.start();
```

Comment:
Could optionally pass in `store.dispatch`, and dispatch inside the startup to save another line of boilerplate. However, this would mean waiting for the redux store is created before running the startup code. Up to you 🦆